nuxt.config.js

```diff
@@ -15,14 +13,10 @@ export default defineNuxtConfig({
       link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }]
     }
   },
-  build: {
-    postcss: {
-      postcssOptions: {
-        plugins: {
-          tailwindcss: {},
-          autoprefixer: {},
-        },
-      },
+  postcss: {
```

Comment:
`postcss` structure has been updated in [latest version](https://nuxt.com/docs/api/configuration/nuxt-config#postcss). We need this to integrate Tailwind :)